### PR TITLE
Added patrol boat

### PR DESCRIPTION
**Pull request by Dzierzan**

### CODE REVIEW

File: mods/hv/rules/ships.yaml [hunk 1]
```diff
@@ -47,6 +47,58 @@ BOAT3:
...
+		Cost: 400
+	Tooltip:
+		Name: Patrol boat
+		GenericName: Boat
+	Buildable:
+		Queue: Ship
+		BuildAtProductionType: Ship
+		Prerequisites: ~harbor.yi
+		BuildPaletteOrder: 40
+		Description: Light patrol boat.\nExcellent against air units.\nNot effective against other units.\nCan detect submarines.\n
```
Comment (Mailaender) [inline mods/hv/rules/ships.yaml:63]: Aren't submarines the natural counter to submarines?

File: mods/hv/rules/ships.yaml [hunk 2]
```diff
@@ -47,6 +47,58 @@ BOAT3:
 	WithMuzzleOverlay:
 		IgnoreOffset: True
 
+AABOAT:
```
Comment (Mailaender) [inline mods/hv/rules/ships.yaml:50]: I would name it differently as I think its role could change in the near future.

File: mods/hv/rules/ships.yaml (hunk 1 above)
Comment (Dzierzan) [reply mods/hv/rules/ships.yaml:63]: Well, I can make submarines detect other submarines and this unit won't do so. In OpenRA RA nearly everything can detect submarine at close range, apart from Battleships.

File: mods/hv/rules/ships.yaml (hunk 2 above)
Comment (Dzierzan) [reply mods/hv/rules/ships.yaml:50]: Sure.

File: mods/hv/rules/ships.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/hv/rules/ships.yaml:63]: I mean it can detect them, but it does only little damage which I think is confusing.

Comment (Dzierzan) [reply mods/hv/rules/ships.yaml:63]: Well, you are supposed to use cheap patrol boats and submarines together.

**Pull request merged**